test/test_binary_ufuncs.py

```diff
@@ -1359,100 +1359,109 @@ def test_rdiv(self, device, dtype):
         self.assertEqual(y, z, exact_dtype=False)
 
     @dtypes(*torch.testing.get_all_fp_dtypes(include_bfloat16=False))
-    def test_fmod_by_zero_float(self, device, dtype):
-        # check floating-point tensor fmod to zero is nan on both CPU and GPU
-        x = make_tensor((10, 10), device=device, dtype=dtype, low=-9, high=9)
-        zero = torch.zeros_like(x)
-
-        self.assertTrue(torch.all(x.fmod(0.0).isnan()))
-        self.assertTrue(torch.all(x.fmod(zero).isnan()))
-        # out
-        out = torch.empty(0, device=device, dtype=dtype)
-        torch.fmod(x, zero, out=out)
-        self.assertEqual(out.size(), torch.Size([10, 10]))
-        self.assertTrue(torch.all(out.isnan()))
-        # in-place
-        x.fmod_(zero)
-        self.assertTrue(torch.all(x.isnan()))
+    def test_fmod_remainder_by_zero_float(self, device, dtype):
+        fn_list = [(torch.fmod, lambda x, mod: x.fmod_(mod)),
```

Comment:
This enumeration of functions is cool but I think it can be simplified to:

`(torch.Tensor.fmod_, torch.Tensor.remainder_)`

Reply:
Also, in general we should tuples for iterables. Lists are OK when the iterable has to be mutable.

Reply:
Note that the behavior of returning NaN when the denominator of fmod or remainder is zero is consistent with the C++ standard but inconsistent with NumPy. That's OK. We should prefer the C++ or Python standards where applicable.